recipes/dsistudio/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "dsistudio" %}
+{% set version = "2021.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/frankyeh/DSI-Studio/archive/refs/tags/2021.10.05.zip
+    sha256: 6a9d1d2e249063a3bafc1fa6a99eebe776b35bcb1fd3034c24d290240be6ecfa
+    folder: src
+  - url: https://github.com/frankyeh/DSI-Studio-atlas/archive/refs/tags/2021.10.zip
+    sha256: 0ea27148512608674fe57e6eb10e523787202fb1f4de30d61d6d973e7c7f0367
+    folder: atlas
+  - url: https://github.com/frankyeh/TIPL/archive/refs/tags/2021.10.04.zip
+    sha256: b5926185484543c2067cdd1d20787e0372fd01636632fd5a87a5f358225bf765
+    folder: src/tipl
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - jom                            # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - {{ cdt('mesa-dri-drivers') }}  # [unix]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('expat-devel') }}       # [linux]
+    - zlib                           # [linux]
+    - libglu                         # [linux]
+    - boost-cpp
+  
+  host:
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - qt 5.12.*                                   
+    - libglu                         # [linux]
+    - gcc 9.*                        # [linux]
+    - gxx 9.*                        # [linux]
+    - boost-cpp
+    - zlib
+  
+  run:
+    - qt 5.12.*
+    - libglu                         # [linux]
+    - ucrt                           # [win]      
```

Comment:
```suggestion
  build:
    - {{ compiler('cxx') }}
    - {{ compiler('c') }}
    - jom                            # [win]
    - {{ cdt('mesa-libgl-devel') }}  # [unix]
    - {{ cdt('mesa-dri-drivers') }}  # [unix]
    - {{ cdt('libselinux') }}        # [linux]
    - {{ cdt('libxcb') }}            # [linux]
    - {{ cdt('libxdamage') }}        # [linux]
    - {{ cdt('libxfixes') }}         # [linux]
    - {{ cdt('libxxf86vm') }}        # [linux]
    - {{ cdt('libxext') }}           # [linux]
    - {{ cdt('libxau') }}            # [linux]
    - {{ cdt('expat-devel') }}       # [linux]
    - {{ cdt('mesa-libgl-devel') }}  # [linux]
    - {{ cdt('mesa-dri-drivers') }}  # [linux]
  host:
    - xorg-libxfixes  # [linux]
    - qt                                  
    - libglu                         # [linux]
    - boost-cpp
    - zlib
  run:
    - qt
    - libglu                         # [linux]
    - ucrt                           # [win]      
```

Conda-forge has pinned the qt framework to 5.12 already, so specifying this version is not needed. Compilers should only be listed in the build section.

The compiler versions are also pinned. Will your build not complete with gcc=10?

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl

Reply:
The build should good with gcc > 9
The commit accordingly.